mappings/upheno_custom_mapping_head_100.sssom.tsv

```diff
@@ -0,0 +1,101 @@
+subject_id	subject_label	subject_source	predicate_id	object_id	object_label	object_source	mapping_justification
+HP:0010951	Abnormal third ventricle morphology (HPO)	obo:0010951	semapv:crossSpeciesExactMatch	XPO:0102755	abnormal third ventricle morphology (XPO)	obo:0102755	semapv:LexicalMatching
```

Comment:
- [ ] object_id should always be the ontology (`obo:hp` or `obo:mp`) of the `object_id`
- [ ] Same for subject_id
- [ ] I would like to add `subject_match_field` and `object_match_field` - do you think that would be possible? (https://mapping-commons.github.io/sssom/)